amida-tech/greyscale
# added function to create user on auth service from indaba backend

#### What's this PR do?
Adds a function that creates a user on the auth-service whenever that user is created on indaba

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-385

#### How should this be manually tested?
- Make sure you have the auth-service running
After you have gotten the auth-service running, there are a few ways to test this:
- You can run the `user.integration` test: `node_modules/.bin/mocha test/user.integration.js --bail -t 10000`. Make sure all the tests pass and you can verify that the users were created on the auth-service by looking at the `users` table on the auth-service

- You can run indaba-client, create a project and add users to that project. When you add a user, verify that the user was added to indaba and also the auth service. 

- You can also use postman to hit this end point: `http://localhost:3005/testorg/v0.2/users/`
#### Any background context you want to provide?
Auth service must be running
#### Screenshots (if appropriate):
